builtin/logical/pki/path_fetch.go

```diff
@@ -96,6 +110,37 @@ func (b *backend) pathFetchCertList(req *logical.Request, data *framework.FieldD
 	return logical.ListResponse(entries), nil
 }
 
+func (b *backend) pathFetchReadCAChain(req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	caInfo, err := fetchCAInfo(req)
+	switch err.(type) {
+	case errutil.UserError:
+		return nil,
+			errutil.UserError{Err: fmt.Sprintf("Could not fetch the CA certificate: %s", err)}
+	case errutil.InternalError:
+		return nil,
+			errutil.InternalError{Err: fmt.Sprintf("Error fetching CA certificate: %s", err)}
+	}
+
+	caChain := caInfo.GetCAChain()
+	certs := []byte{}
+	for _, ca := range caChain {
+		block := pem.Block{
+			Type:  "CERTIFICATE",
+			Bytes: ca.Bytes,
+		}
+		certs = append(certs, pem.EncodeToMemory(&block)...)
+	}
+
+	response := &logical.Response{
```

Comment:
>  I think you were thinking about cert/ca which returns a Vault response.
I was! Mea culpa. So this seems fine as-is, but it would be nice to support `cert/ca_chain`. It could just build up the chain rather than attempt to fetch one from the backend, unless this is difficult for a reason I'm not thinking of right now.